dbfit-java/core/src/main/java/dbfit/fixture/SetOption.java

```diff
@@ -0,0 +1,72 @@
+package dbfit.fixture;
```

Comment:
Two points to consider are:
* Trying to keep uniform syntax between Standalone and Flow modes (when it makes sense) so that it's easier to reuse/migrate a test page between modes.
* Compatibility with existing tests.